springfox-bean-validators/src/main/java/springfox/bean/validators/plugins/Validators.java

```diff
@@ -133,16 +132,18 @@ public static boolean annotationMustBeApplied(ModelPropertyContext context, Clas
   }
```

Comment:
### Third condition (or `Case #3`) - 130 line

The system determines the intersection between two sets — annotation groups `@Valid` (equivalent to `@Validated(groups=Default.class)`) or `@Validated(groups=...)` and annotation groups constraints.

If an intersection exists, then the constraint annotation must be applied to this model.